finagle/h2/src/main/scala/com/twitter/finagle/buoyant/h2/netty4/Netty4DispatcherBase.scala

```diff
@@ -187,7 +187,7 @@ trait Netty4DispatcherBase[SendMsg <: Message, RecvMsg <: Message] {
   }
 
   protected[this] val onTransportClose: Throwable => Unit = { e =>
-    log.debug(e, "[%s] transport closed", prefix)
+    log.debug("[%s] transport closed", prefix)
```

Comment:
ideally, we would log on unexpected errors.  i'm not immediately sure if it's possible to get non-connection-related errors here, though.  Perhaps a good compromise is to at least log the exceptions message?